lib/debug/breakpoint.rb

```diff
@@ -8,7 +8,7 @@ class Breakpoint
 
     attr_reader :key
 
-    def initialize cond, command, path, do_enable: true
+    def initialize(cond, command, path, do_enable: true)
```

Comment:
does it related to the main goal?

Reply:
ugh, sorry... This was my IDE doing some autoformatting. I'll revert